src/main/java/io/jenkins/plugins/pipelinegraphview/utils/PipelineStepApi.java

```diff
@@ -22,41 +22,45 @@
 public class PipelineStepApi {
     private static final Logger LOGGER = Logger.getLogger(PipelineStepApi.class.getName());
     private transient WorkflowRun run;
-    private transient FlowNode node;
     private final String nodeId;
 
+    private static PipelineStepVisitor builder;
+    private static Object mutex = new Object();
 
-    public PipelineStepApi(WorkflowRun run, String nodeId) throws java.io.IOException {
+    public PipelineStepApi(WorkflowRun run, String nodeId) {
```

Comment:
node id is assigned but never accessed is there a reason for that?

Reply:
An artefact of the was I was trying to get the steps before.